tests/drivers/regulator/voltage/boards/nrf52840dk_nrf52840_npm1100.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	resources: resources {
+		compatible = "test-regulator-voltage";
+		regulators = <&npm6001_ek_buck>;
```

Comment:
```suggestion
		regulators = <&npm1100_ek_buck>;
```

Reply:
oops, not sure how that ended up here, test was passing locally.